opendebates/templates/opendebates/top_archive.html

```diff
@@ -0,0 +1,53 @@
+{% extends "opendebates/base.html" %}
+{% block content %}
+
+{% load i18n l10n %}
+
+<div class="row">
+  <div class="col-sm-8 col-sm-offset-2">
+    {{ category.title|safe }}
+    <div class="row idea-list">
+      <div class="winners">
+        {% for idea in submissions %}
```

Comment:
Note: everything inside this for loop is very very similar to `templates/opendebates/snippets/idea.html` -- I'm not happy about the duplication, but there are enough differences (e.g. ordinal rank instead of vote button, no share section, no hyperlinks) that I thought I'd better not try to share code, especially since that idea.html template is already enormously complex and branchy.